tests/lib/linter/linter.js

```diff
@@ -3347,6 +3359,44 @@ var a = "test2";
                         message: "Unused eslint-disable directive (no problems were reported).",
                         line: 1,
                         column: 1,
+                        fix: {
+                            range: [0, 20],
+                            text: " "
+                        },
+                        severity: 1,
+                        nodeType: null
+                    }
+                ]
+            );
+        });
+
+        it("reports problems for partially unused eslint-disable comments (in config)", () => {
```

Comment:
Was there anything in particular that should be tested at this level? I put in some unit-level ones into `apply-disable-directives.js` around commas, spacing, and ordering of rule IDs.

Reply:
Marking as resolved since there's been a review since the comment.